dockstore-webservice/src/main/java/io/dockstore/webservice/resources/EntryResource.java

```diff
@@ -381,6 +385,31 @@ public List<VersionVerifiedPlatform> getVerifiedPlatforms(@Parameter(hidden = tr
         return verifiedVersions;
     }
 
+    @GET
+    @Path("/{entryId}/syncing")
+    @UnitOfWork
+    @Operation(operationId = "isSyncing", description = "Is this entry being updated automatically when the source repository changes?", security = @SecurityRequirement(name = JWT_SECURITY_DEFINITION_NAME))
```

Comment:
In the spirit of my other comments, I'd say "Is Dockstore being notified when the source repository changes".